examples/google

```diff
@@ -0,0 +1,65 @@
+#Install Vertex AI
```

Comment:
You can call this file `google-gemini.ipynb` and use actual format of the notebook. I think it will be helpful to have each step described that way.

Reply:
Hi Oleksandr Yaremchuk, thanks for reviewing my PR. I added to the comment for more specific instructions to set up a notebook view this link https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb.  Is that what you would like or put the link and copy and paste some of the steps?